noxfile.py

```diff
@@ -33,6 +33,9 @@
 OSX_BUNDLE_IDENTIFIER = 'com.backblaze.b2'
 OSX_BUNDLE_ENTITLEMENTS = 'contrib/macos/entitlements.plist'
 
+WINDOWS_TIMESTAMP_SERVER = 'http://timestamp.digicert.com'
```

Comment:
this link doesn't work AND it's http?

Reply:
The command was provided by @ooola. I guess it uses https://www.ietf.org/rfc/rfc3161.txt and TSP over HTTP (chapter 3.4)

Reply:
HTTPS doesn't work with TSP - tested
![image](https://user-images.githubusercontent.com/63924630/132986089-71a084ee-9b50-424a-ac04-5c8abe5b1e40.png)